package.json

```
@@ -22,7 +22,9 @@
     "test": "vitest --project unit",
     "test:bun": "bun test --bail",
     "test:e2e": "vitest --project e2e",
-    "test:leaks": "cross-env \"LEAK_TEST=1\" jest --no-watchman --runInBand --detectOpenHandles --detectLeaks --logHeapUsage"
+    "test:leaks": "cross-env \"LEAK_TEST=1\" jest --no-watchman --runInBand --detectOpenHandles --detectLeaks --logHeapUsage",
+    "build:alpha": "yarn build && yarn pin-workspace-deps",
```

Each package has a `prepack` script which will build only the package being released. So building all packages is completely unnecessary.

```suggestion
    "build:alpha": "yarn pin-workspace-deps",
```